python/ikarus/io/io.hh

```diff
@@ -1,22 +1,26 @@
 // SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
 // SPDX-License-Identifier: LGPL-3.0-or-later
 
-#include <config.h>
-
+/**
+ * \file io.hh
+ * \brief Additional python bindings for submodule io
+ */
 #include "../pythonhelpers.hh"
 
-#include <dune/python/common/typeregistry.hh>
 #include <dune/python/pybind11/eigen.h>
 #include <dune/python/pybind11/operators.h>
 #include <dune/python/pybind11/pybind11.h>
 
 #include <ikarus/io/vtkdatatag.hh>
 
-PYBIND11_MODULE(_io, m) {
+void addBindingsToIO() {
   namespace py = pybind11;
   using namespace pybind11::literals;
+  using namespace Ikarus;
+
+  auto io = pybind11::module::import("ikarus.io");
```

Comment:
Does this always work because the might not be defined yet?

Maybe better like ther material module: "auto materials = m.def_submodule("materials", "This is the submodule for materials in Ikarus");"

Reply:
This is what i first tried, but it didn't work. 
From my understanding we define the _io and the other modules with the cmake stuff and the init.py. 
This seems to happen first. When you then readd the module (with m.add_submodule) then nothing gets added. 

So this seems to work everytime, when we have this init.py and some c++ definitions.  With material its different, as we don't have this init.py and it can be added directly

I hope that made sense :D